sys-process/dcron/dcron-4.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cron toolchain-funcs systemd
+
+DESCRIPTION="Dillon's lightweight and minimalist cron daemon"
+HOMEPAGE="
+	https://github.com/ptchinster/dcron
+	http://www.jimpryor.net/linux/dcron.html
+	https://apollo.backplane.com/FreeSrc/
```

Comment:
These probably aren't relevant anymore?

Reply:
You do make a point, but I feel like since GH is the current active upstream, that should hold whatever history they want to tell regarding this package. Going to www.jimpryor.net does indeed show where the program is coming from, but it doesn't lead you downloading the _current_ version so it's also a bit deceptive.

No hard feelings on that one either way, make your own decision.